central/policy/service/validator.go

```diff
@@ -341,13 +341,21 @@ func (s *policyValidator) compilesForDeployTime(policy *storage.Policy, options
 		return errors.Wrap(err, "policy configuration is invalid for deploy time")
 	}
 	if booleanpolicy.ContainsRuntimeFields(policy) {
-		return errors.New("deploy time policy cannot contain runtime fields")
+		return errors.New("deploy time policy cannot contain runtime criteria")
 	}
 	return nil
 }
 
 func (s *policyValidator) compilesForRunTime(policy *storage.Policy, options ...booleanpolicy.ValidateOption) error {
-	// Runtime policies must contain one or more runtime fields, but can have deploy time fields as well
+	// Runtime policies must contain one category of runtime criteria, but can have deploy time criteria as well
+	if !booleanpolicy.ContainsRuntimeFields(policy) {
+		return errors.New("A run time policy must contain at least one runtime specific criteria from the process, network flow, audit log events or Kubernetes events categories")
```

Comment:
nit: `run time` (two words) and `runtime` (one word) in same sentence.
We have it as oneword in the app input.

(also, no Oxford comma, but nobody's perfect)

```suggestion
		return errors.New("A runtime policy must contain at least one runtime specific criteria from the process, network flow, audit log events, or Kubernetes events categories")
```